tests/paasta_cli/test_cmds_validate.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mock import patch
+from pytest import raises
+from StringIO import StringIO
+
+from paasta_tools.paasta_cli.cmds.validate import get_schema
+from paasta_tools.paasta_cli.cmds.validate import validate_schema
+from paasta_tools.paasta_cli.cmds.validate import paasta_validate
+from paasta_tools.paasta_cli.utils import PaastaValidateMessages
+
+
+@patch('paasta_tools.paasta_cli.cmds.validate.guess_service_name')
+@patch('paasta_tools.paasta_cli.cmds.validate.validate_all_schemas')
+def test_paasta_validate_calls_everything(
+        mock_validate_all_schemas,
+        mock_guess_service_name
+):
+    # Ensure each check in 'paasta_validate' is called
+
+    mock_guess_service_name.return_value = 'servicedocs'
+    paasta_validate(None)
+
+    assert mock_validate_all_schemas.called
+
+
+def test_get_schema_marathon_found():
+    get_schema('marathon')
```

Comment:
you're making no assertions here other than that it doesn't throw an exception - can you verify that the thing returned is what you expect?